supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -1,6 +1,47 @@
 [[formatting]]
 = Formatting
 
+[[commands-in-code-blocks]]
+== Commands in code blocks
+
+Use a single command per code block for each procedure step. This approach helps with readability and makes it possible for the copy button in code blocks to work correctly.
+
+By default, use bold formatting for commands in code blocks to visually distinguish the commands from their lead-in sentences and from the command prompts and example outputs. For consistency, use bold formatting for commands even when no command output is shown in the code block.
+
+[NOTE]
+====
+To apply formatting in a code block, you must use the `quotes` link:https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/[AsciiDoc substitution].
+====
+
+.Example AsciiDoc: A command and its output in a code block
+
+  Verify that the `libvirt` default network is active and configured to start automatically:
```

Comment:
My only feedback is whether this example should actually be shown as a bullet or numbered step. But I don't think that's important enough to hold up the PR over since it has 2 other approvals. Just something to consider if we should adjust in a separate PR or not!